modules/gitops-release-notes-1-1.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-1_{context}"]
+= Release notes for {gitops-title} 1.1
+
+[id="gitops-technology-preview-features-1-1_{context}"]
+== GitOps Technology Preview features
```

Comment:
Drop the GitOps, does not seem necessary.

Reply:
Done